.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+      "browser": true,
+      "es2021": true
+    },
+    
+    "extends": [
+      "plugin:prettier/recommended",
+      "eslint:recommended",
+      "plugin:@typescript-eslint/recommended",
+      "prettier"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "ecmaVersion": "latest",
+      "sourceType": "module"
+    },
+    
+    "plugins": ["@typescript-eslint", "promise"],
+    "rules": {
+        "prettier/prettier": [
+          "error",
+          {
```

Comment:
it will be better when you add this rules also : 
 {
        "tabWidth": 2,
        "printWidth": 80,
        "useTabs": false,
        "semi": true,
        "singleQuote": true,
        "trailingComma": 'all',
        "arrowParens": 'avoid',
        "endOfLine": 'lf',
      },